api/src/lib-http.js

```diff
@@ -0,0 +1,50 @@
+// This library file contains the interface to the http service.
```

Comment:
Would deploy-script-support be the right place for this?

Reply:
> Would deploy-script-support be the right place for this?

Asbolutely!

Importing it there and deduping (currently part of dapp-fungible-faucet et al) would be a laudable cleanup.